runtime-modules/forum/src/lib.rs

```diff
@@ -172,48 +166,6 @@ pub trait StorageLimits {
 
     /// Maximum total of all existing categories
     type MaxCategories: Get<u64>;
-
-    /// Maximum number of poll alternatives
-    type MaxPollAlternativesNumber: Get<u64>;
```

Comment:
Should be also removed from `!parameter_types` in `runtime-modules/forum/src/mock.rs`